stablehlo/dialect/StablehloOps.td

```diff
@@ -1980,7 +1980,7 @@ def StableHLO_CustomCallOp: StableHLO_Op<"custom_call",
   }];
 
   let arguments = (ins
-    Variadic<HLO_TensorOrTokenOrTuple>:$operands,
+    Variadic<HLO_TensorOrTokenOrTuple>,
```

Comment:
Let's call these `inputs` as well?

Reply:
good catch - this was a test to see if this argument's getter was used anywhere. In StableHLO it is not, probably a different story for MHLO. Will rename.